docs/content/concepts/04-accepting-input.md

```diff
@@ -39,32 +39,33 @@ User input comes in three forms:
 
 ## Validating User Input
 
```

Comment:
Did a quick sweep through this doc to remove references to decorators & fix a link to AJV.